test/zaim.js

```diff
@@ -286,6 +286,26 @@ describe("private method suite", function() {
       yy = (ymd.getYear() % 1900) + 1900,
       mm = ymd.getMonth() + 1,
       dd = ymd.getDate();
-    expect(zaim._getCurrentDate()).to.equal(yy + "-" + mm + "-" + dd);
+    expect(zaim.getCurrentDate()).to.equal(yy + "-" + mm + "-" + dd);
+  });
+  it("should return valid formatted date", function() {
+    var zaim = new Zaim({
+        consumerKey: "consumerKey",
+        consumerSecret: "consumerSecret",
+        accessToken: "accessToken",
+        accessTokenSecret: "accessTokenSecret"
+      });
+    expect(zaim.formatDate(new Date(2018, 10, 24))).to.equal("2018-11-24");
+  });
+  it("should throw an error with wrongly-formatted string", function() {
```

Comment:
Nit: Can we use arrow function? 👀 
```js
  it("should throw an error with wrongly-formatted string", () => {
```

Reply:
Sure. It looks much more sophisticated!  I think we can also use `const`. I've just committed these changes.